financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsManifestRepository.kt

```diff
@@ -122,6 +124,11 @@ internal interface FinancialConnectionsManifestRepository {
         selectedAccounts: List<String>
     ): FinancialConnectionsSessionManifest
 
+    suspend fun disableNetworking(
```

Comment:
could you add a comment on this method?

Reply:
https://github.com/stripe/stripe-android/pull/6185/commits/3a328e18de19de89cd2602c136c396f6a61c0bac